packages/fieldset/src/FormGroupMixin.js

```diff
@@ -392,7 +392,7 @@ export const FormGroupMixin = dedupeMixin(
        */
       removeFormElement(...args) {
         super.removeFormElement(...args);
-        this.validate();
+        this.validate({ clearCurrentResult: true });
```

Comment:
We should also clear in `addFormElement ` I think

Reply:
done :)